### ISSUE TITLE
Unable to load DLL 'libSkiaSharp' on Linux using version 1.60.0

### ISSUE THREAD

**Issue by frliran** [mono/SkiaSharp#509]
### Description

I have a .Net Core 2.0 WebAPI Project which uses a class library which references SkiaSharp 1.60.0 NuGet. When I publish my project to Ubuntu-16.04-x64 the file "libSkiaSharp.so" is missing and I'm getting an exception: "Unable to load DLL 'libSkiaSharp'". I came across the same issue here #288 but it didn't fix it.

I tried using "Avalonia.Skia.Linux.Natives 1.57.1.4" but it only works with Skia's 1.57.1 version and I want to use the latest 1.60.0.

Can you please tell me if and how this issue was solved?

### Actual Behavior

Gets an error "Unable to load DLL 'libSkiaSharp'"

### Basic Information

- Version with issue:  1.60.0
- IDE:  Visual Studio 2017
- Platform Target Frameworks:
  - Linux:  Ubuntu 16.04

**Comment by mattleibow** [comment]
Due to the number of Linux distros, we don't provide a native binary _in_ the NuGet package. But, we do host one on the release notes:
https://github.com/mono/SkiaSharp/releases/tag/v1.60.0

And a direct link to the native file: [libSkiaSharp.so](https://github.com/mono/SkiaSharp/releases/download/v1.60.0/libSkiaSharp.so)

**Comment by VesaVainio** [comment]
Just commenting, in case this will benefit somebody else who stumbles on the same issue. In my case we are using a third-party library that depends on SkiaSharp. Our application is using .NET Core 3.1, and it's running in Docker containers on top of Ubuntu 16.04.

In our troubleshooting case, we had to do BOTH of the following steps to get SkiaSharp to work. I tested both of them individually, and neither one alone solves it, but both of them together does solve it.

1. Adding the following line to dockerfile, to install necessary libraries inside the container:
`RUN apt-get update && apt-get install -y libfontconfig1`
2. Including the SkiaSharp.NativeAssets.Linux nuget package (version 1.68.3) to the .NET Core project where SkiaSharp is used.

**Comment by mattleibow** [comment]
@VesaVainio thanks for this. Yes, fontconfig is a requirement for that package. I think I must update the description or some readme to say that it needs to be installed.

But also just for info, there is also a nuget package SkiaSharp.NativeAssets.Linux.NoDependencies that does not have this requirement. But, it also does not have the same level of font management.

**Comment by VesaVainio** [comment]
@mattleibow just to clarify the background for how we stumbled on this problem: we were using a third party library, and SkiaSharp comes to us as a dependency for that package. (The library in question is Syncfusion, Syncfusion.XlsIORenderer.Net.Core nuget package.)

Everything worked fine in local Windows development environments, and we only encountered the problem when deploying the software to a containerized environment that runs on top of Linux. That's the first time we actually became aware that we are using some library called SkiaSharp.

I think the fundamental problem is that when you are using NuGet packages, you pretty much just think that once installed, they will work.

I'm not sure what would be the optimal solution to this problem. Seems that the NuGet dependency resolution is centered around the frameworks and versions of packages, not on OS. Should it be so that if SkiaSharp publishes a NuGet package, it should work as-is even on top of Linux? Or should it be so that Suncfusion should include enough packages to make sure that their dependencies work on Linux? Not sure...
